src/test/scripts/helm_install.sh

```diff
@@ -81,7 +81,7 @@ then
 fi
 
 # Run the chart's tests
-helm test "$PRODUCT_RELEASE_NAME" --logs -n "${TARGET_NAMESPACE}"
```

Comment:
The `--logs` will now no longer work, since the pod will be deleted as soon as the test passes.

Note that failed tests should still have their logs captured by the "log download" task.